hieradata/vagrant/clients/oracle.updates.jenkins.io.yaml

```
@@ -0,0 +1,12 @@
+---
+lvm::volume_groups:
+  data:
+    physical_volumes:
+      - /dev/loop0
+    logical_volumes:
+      updates_logs:
+        size: 600G
```

```suggestion
        size: 15M
```

You can keep this value low in vagrant